pkg/app/piped/controller/planner.go

```diff
@@ -189,7 +189,7 @@ func (p *planner) Run(ctx context.Context) error {
 	}
 
 	defer func() {
-		controllermetrics.UpdateDeploymentStatus(p.deployment.Id, p.doneDeploymentStatus, p.deployment.Kind, p.deployment.PlatformProvider)
+		controllermetrics.UpdateDeploymentStatus(p.deployment, p.deployment.Status)
```

Comment:
@kentakozuka should it p.doneDeploymentStatus instead? 🤔

Reply:
Thanks for pointing out!
Let me fix it 👍

Reply:
Ah, sorry... :cry:

Reply:
No prob. It was my fault too.